tests/Operation/RenameCollectionFunctionalTest.php

```diff
@@ -25,18 +24,8 @@ public function setUp(): void
         parent::setUp();
 
         $this->toCollectionName = $this->getCollectionName() . '.renamed';
-        $operation = new DropCollection($this->getDatabaseName(), $this->toCollectionName);
-        $operation->execute($this->getPrimaryServer());
-    }
-
-    public function tearDown(): void
-    {
-        if (! $this->hasFailed()) {
-            $operation = new DropCollection($this->getDatabaseName(), $this->toCollectionName);
-            $operation->execute($this->getPrimaryServer());
-        }
-
-        parent::tearDown();
+        $this->dropCollection($this->getDatabaseName(), $this->toCollectionName);
+        $this->dropCollection($this->getDatabaseName(), $this->getCollectionName());
```

Comment:
I think I missed this earlier, but I noticed you're passing explicit arguments here instead of `null, null`. I realize it's more verbose but I actually think it makes these lines more readable.

If you agree, feel free to apply this everywhere and remove the "default on null" logic. If you prefer the "default on null" logic, then I'd suggest revising this to make it consistent with other tests.

Reply:
I removed the default `null`. That's better for readability.